SwiftCairo-App/Networking/Response/ServerResponse.swift

```diff
@@ -0,0 +1,13 @@
+//
+//  ServerResponse.swift
+//  SwiftCairo-App
+//
+//  Created by Mohammed Elnaggar on 2/22/20.
+//  Copyright © 2020 abdelrahman mohamed. All rights reserved.
+//
+
+import Foundation
+
+enum ServerResponse<T> {
+    case success(T), failure(Error)
```

Comment:
Why do we need this? since there's Result now so no need for it

Reply:
yeah, I'll replace it with AFResult